assets/javascript/game.js

```diff
@@ -0,0 +1,37 @@
+
+
+var result = 0;
+var butt1;
+$(document).ready(function () {
+    $(".display").html(Math.floor(Math.random() * 120) + 19); 
+    $("#gem1").on("click", function () {
```

Comment:
A variable for the score to reach needs to be declared so that when the user reaches the score or passes it, they lose or win